examples/flows/evaluation/eval-flow-for-multi-turn/README.md

```diff
@@ -0,0 +1,47 @@
+# Evaluation flow for single turn:
+
+This evaluation flow will evaluate the Q&A systems by leveraging the state-of-the-art Large Language Models (LLM) to measure the quality of the responses. Utilizing GPT and GPT embedding model to assist with measurements aims to achieve a high agreement with human evaluations compared to traditional mathematical measurements.
+
+## What you will learn
+
+This evaluation flow allows you to assess and evaluate your model with the LLM-assisted metrics:
+
+* grounding: Measures whether the answer follows logically from the information contained in the context based on provided answer and context.
+
+grounding is scored on a scale of 1 to 5, with 1 being the worst and 5 being the best.
+
+* answer_relevancy: Measure whether the answer is relevancy to the question based on provided question, context and answer.
+
+answer_relevancy is scored on a scale of 1 to 5, with 1 being the worst and 5 being the best. 
+
+* answer_quality: Measures the answer quality for each of the following factors based on provided question and answer: 
```

Comment:
Should the metric be renamed as `conversation_quality`?

Reply:
Change to conversation_quality